m/mink-selenium2-driver/minkseleniumdriver_ubi8.5.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+# ----------------------------------------------------------------------------
+# Package          : minkseleniumdriver
+# Version          : latest
+# Source repo      : https://github.com/minkphp/MinkSelenium2Driver.git
+# Tested on        : UBI 8.5
+# Language         : PHP
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Ankit Paraskar <Ankit.Paraskar@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+# Variables
+
+PACKAGE_NAME=minkseleniumdriver
+PACKAGE_URL=https://github.com/minkphp/MinkSelenium2Driver.git
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=v6.2.0
+
+yum install -y git curl php php-curl php-dom php-mbstring php-json php-gd php-pecl-zip
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && php composer-setup.php --install-dir=/bin --filename=composer
+php composer-setup.php --install-dir=/usr/local/bin --filename=composer
+yum install unzip -y
+
+HOME_DIR=$(pwd)
+echo $HOME_DIR
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+    exit 1
+fi
+
+
+
+cd "$HOME_DIR"/$PACKAGE_NAME || exit
+git fetch --tags
+latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
what is the version that was requested? looks like we are defining one but testing another one or am I missing something?

Reply:
scripted to pick up the latest code from git repo but as i understand we have hardcoded dependency on specific version. Script is update accordingly also will modify other scripts